cypress/integration/ContextMenu.spec.ts

```diff
@@ -133,21 +133,21 @@ describe('ContextMenu', () => {
       });
 
       it('should close only the focused submenu when pressing left arrow key', () => {
-        cy.findByText('Bookmarks →').type('{enter}');
-        cy.findByText('Modulz →').type('{enter}');
-        cy.findByText('Stitches').type('{leftarrow}').should('not.exist');
+        cy.findByText('Bookmarks →').focus().type('{enter}');
+        cy.findByText('Modulz →').focus().type('{enter}');
+        cy.findByText('Stitches').focus().type('{leftarrow}').should('not.exist');
         cy.findByText('Modulz →').should('be.visible');
         cy.findByText('New Window').should('be.visible');
       });
 
       it('should close all menus when pressing escape, enter or space key on any item', () => {
         // Test close on root menu
-        cy.findByText('Bookmarks →').type('{esc}').should('not.exist');
+        cy.findByText('Bookmarks →').focus().type('{esc}').should('not.exist');
 
         // Reopen menu and test keys from within the submenu
         cy.findByText('Right Click Here').rightclick();
-        cy.findByText('Bookmarks →').type('{enter}');
-        cy.findByText('Inbox').type('{esc}').should('not.exist');
+        cy.findByText('Bookmarks →').focus().type('{enter}');
+        cy.findByText('Inbox').focus().type('{esc}').should('not.exist');
```

Comment:
there were a bunch of failing tests on CI because `.type` API expects the element to have `tabindex` https://docs.cypress.io/api/commands/type#Input-Textarea

triggering focus ensures `tabindex="0"`.